pkg/agent/apiserver/apiserver.go

```diff
@@ -75,6 +76,7 @@ func (s *agentAPIServer) Run(stopCh <-chan struct{}) error {
 
 func installHandlers(aq agentquerier.AgentQuerier, npq querier.AgentNetworkPolicyInfoQuerier, seipq querier.ServiceExternalIPStatusQuerier, s *genericapiserver.GenericAPIServer) {
 	s.Handler.NonGoRestfulMux.HandleFunc("/loglevel", loglevel.HandleFunc())
+	s.Handler.NonGoRestfulMux.HandleFunc("/multicaststats", multicast.HandleFunc(aq))
```

Comment:
If the data is per pod, maybe use "/podmulticaststats" as the path

Reply:
updated.